controllers/task.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2015-2016, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const express = require('express');
+const lighthouseLib = require('../lib/lighthouse');
+const router = express.Router(); // eslint-disable-line new-cap
+
+/**
+ * POST /task/lighthouse/:pwaid
+ *
+ * Fetch and store a lighthouse report for the pwaid
+ */
+router.post('/lighthouse/:pwaid', (req, res, next) => {
```

Comment:
Yes, I think so, I will check tomorrow.

On Sep 6, 2016 7:59 PM, "André Cipriani Bandarra" <notifications@github.com>
wrote:

> In controllers/task.js
> <https://github.com/GoogleChrome/gulliver/pull/103#discussion_r77685245>:
>
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +'use strict';
> > +
> > +const express = require('express');
> > +const lighthouseLib = require('../lib/lighthouse');
> > +const router = express.Router(); // eslint-disable-line new-cap
> > +
> > +/**
> > + * POST /task/lighthouse/:pwaid
> > + *
> > + * Fetch and store a lighthouse report for the pwaid
> > + */
> > +router.post('/lighthouse/:pwaid', (req, res, next) => {
>
> Is there any way we can only allow AppEngine to start tasks?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/gulliver/pull/103/files/713ff846b26b93ee7662f3df36cf048980d1b84d#r77685245>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1O_HDGKnes4RWrssAp0IVUSBqeJlu9ks5qnahRgaJpZM4Jz2iW>
> .
>